newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -291,6 +292,7 @@ const VariablesList = ({ onComputeAllVariableNames, ...props }: Props) => {
     props.historyHandler
       ? props.historyHandler.saveToHistory()
       : setHistory(saveToHistory(history, props.variablesContainer));
+    if (props.onVariablesUpdated) props.onVariablesUpdated();
```

Comment:
Should the method be renamed "onChange" ?

Reply:
I wasn't sure, it felt to me like `onChange` is really generic

Reply:
Sorry, I was talking about the method `_saveToHistory`